Microsoft.Toolkit.Uwp.UI.Controls/Graph/ProfileCard/DisplayModeConverter.cs

```diff
@@ -0,0 +1,39 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Graph
+{
+    internal class DisplayModeConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, string language)
+        {
+            if (value is ViewType && parameter != null)
```

Comment:
Can't you use a function binding for this? Should be faster and one less file to maintain.

Reply:
@azchohfi I think to use x:Bind they'd have to add x:Class to the ResourceDictionary.  The default templated control template in VS doesn't set this up for you.

Reply:
If keeping as a converter, I think it could be useful to generalize and move this to our other converters in Microsoft.Toolkit.Uwp.UI.Converters.  Like `EqualityToVisibilityConverter`.  Just remove the is typecheck (make it a null check) and compare the ToString result on the value and parameter as you're doing (you can combine the if statements).

Reply:
@michael-hawker Let's ignore this as now the DisplayModeConverter is replaced by DataTemplate